platform/android/export/export.cpp

```diff
@@ -2415,13 +2499,17 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			String remote_plugins_binaries = get_plugins_binaries(BINARY_TYPE_REMOTE, enabled_plugins);
 			String custom_maven_repos = get_plugins_custom_maven_repos(enabled_plugins);
 			bool clean_build_required = is_clean_build_required(enabled_plugins);
+			String version_code = itos(p_preset->get("version/code"));
```

Comment:
Move these 2 lines right below "String package_name = get_package_name" to keep the ordering and organization consistent.